src/apps/common/submodules/portListing/portListing.js

```diff
@@ -100,15 +100,21 @@ define(function(require) {
 						}
 					};
 				}),
-				tabs = _.map(self.appFlags.portListing.tabs, function(tab) {
-					return {
-						text: self.i18n.active().portListing.tabs[tab],
-						id: tab,
-						menus: [{
-							tabs: subTabs
-						}]
-					};
-				}),
+				tabs = _
+					.chain(self.appFlags.portListing.tabs)
+					.filter(function(tab) {
+						return tab !== 'agent' || monster.util.isSuperDuper() || monster.config.whitelabel.port.authority === self.accountId;
```

Comment:
That whole path won't necessarily be set (`monster.config.whitelable` will for sure).